### PR TITLE
run upstream velero ci against downstream

### PR DESCRIPTION
**Pull request by weshayutin**
seeing if running upstream ci against our downstream copy is valuable. /hold for now.

### CODE REVIEW

**Comment by weshayutin** [comment]
oops.  git commit -m "looks like we need `make update` before make ci`" -s

File: .travis.yml [hunk 1]
```diff
@@ -46,6 +46,11 @@ script:
 
 jobs:
   include:
+  - stage: TEST - make ci
+    script: 
+    - | 
+      make update
+      make ci
```
Comment (kaovilai) [inline .travis.yml]:
```suggestion
      make update ci
```

**Review by kaovilai — CHANGES_REQUESTED** [review]

**Review by dymurray — APPROVED** [review]
LGTM

**Review by kaovilai — APPROVED** [review]

**Comment by kaovilai** [comment]
There's 8 commits here. Could you rebase/squash?

**Comment by weshayutin** [comment]
@kaovilai there is a squash and merge option.. why not just use that?

**Comment by kaovilai** [comment]
I'm ok with that usually. I'm concerned here that the commits in this PR include commits you did not author. Double check that you're using the right branch for this PR to merge into.
> [dymurray](https://github.com/dymurray) and others added 5 commits [last month](https://github.com/openshift/velero/pull/230#commits-pushed-43b816d)

**Comment by weshayutin** [comment]
@kaovilai ya. it just needed to be rebased and squashed..

**Review by sseago — APPROVED** [review]

**Pull request closed without merging**